Documentation/ceph-fs-mirror-crd.md

```diff
@@ -39,8 +39,61 @@ If any setting is unspecified, a suitable default will be used automatically.
+  However, if you want to enable mirroring of multiple filesystems, you would have to have **one Secret per filesystem**.
 * `placement`: The cephfs-mirror pods can be given standard Kubernetes placement restrictions with `nodeAffinity`, `tolerations`, `podAffinity`, and `podAntiAffinity` similar to placement defined for daemons configured by the [cluster CRD](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/cluster.yaml).
 * `annotations`: Key value pair list of annotations to add.
 * `labels`: Key value pair list of labels to add.
 * `resources`: The resource requirements for the cephfs-mirror pods.
 * `priorityClassName`: The priority class to set on the cephfs-mirror pods.
+
+### Configuring mirroring peers
+
+On an external site you want to mirror with, you need to create a bootstrap peer token.
+The token will be used by one site to **pull** images from the other site.
+The following assumes the name of the pool is "test" and the site name "europe" (just like the region), so we will be pulling images from this site:
+
+```console
+external-cluster-console # ceph fs snapshot mirror peer_bootstrap create myfs2 client.mirror europe
+{"token": "eyJmc2lkIjogIjgyYjdlZDkyLTczYjAtNGIyMi1hOGI3LWVkOTQ4M2UyODc1NiIsICJmaWxlc3lzdGVtIjogIm15ZnMyIiwgInVzZXIiOiAiY2xpZW50Lm1pcnJvciIsICJzaXRlX25hbWUiOiAidGVzdCIsICJrZXkiOiAiQVFEVVAxSmdqM3RYQVJBQWs1cEU4cDI1ZUhld2lQK0ZXRm9uOVE9PSIsICJtb25faG9zdCI6ICJbdjI6MTAuOTYuMTQyLjIxMzozMzAwLHYxOjEwLjk2LjE0Mi4yMTM6Njc4OV0sW3YyOjEwLjk2LjIxNy4yMDc6MzMwMCx2MToxMC45Ni4yMTcuMjA3OjY3ODldLFt2MjoxMC45OS4xMC4xNTc6MzMwMCx2MToxMC45OS4xMC4xNTc6Njc4OV0ifQ=="}
+```
+
+For more details, refer to the official ceph-fs mirror documentation on [how to create a bootstrap peer](https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers).
+
+When the peer token is available, you need to create a Kubernetes Secret.
+Our `europe-cluster-peer-fs-test-1` will have to be created manually, like so:
+
+```console
+$ kubectl -n rook-ceph create secret generic "europe-cluster-peer-fs-test-1" \
+--from-literal=token=eyJmc2lkIjogIjgyYjdlZDkyLTczYjAtNGIyMi1hOGI3LWVkOTQ4M2UyODc1NiIsICJmaWxlc3lzdGVtIjogIm15ZnMyIiwgInVzZXIiOiAiY2xpZW50Lm1pcnJvciIsICJzaXRlX25hbWUiOiAidGVzdCIsICJrZXkiOiAiQVFEVVAxSmdqM3RYQVJBQWs1cEU4cDI1ZUhld2lQK0ZXRm9uOVE9PSIsICJtb25faG9zdCI6ICJbdjI6MTAuOTYuMTQyLjIxMzozMzAwLHYxOjEwLjk2LjE0Mi4yMTM6Njc4OV0sW3YyOjEwLjk2LjIxNy4yMDc6MzMwMCx2MToxMC45Ni4yMTcuMjA3OjY3ODldLFt2MjoxMC45OS4xMC4xNTc6MzMwMCx2MToxMC45OS4xMC4xNTc6Njc4OV0ifQ==
+```
+
+Rook will read a `token` key of the Data content of the Secret.
+
+You can now inject the fsmirror CR:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemMirror
+metadata:
+  name: my-fs-mirror
+  namespace: rook-ceph
+spec:
+  peers:
+    secretNames:
+      - "europe-cluster-peer-pool-test-1"
+```
+
+You can add more filesystems, for this just repeat the above and change the "token" value of the Kubernetes Secret.
```

Comment:
All the filesystems would be in the same cluster, right? Is it possible to connect to filesystems in different clusters?
```suggestion
You can add more filesystems by repeating the above and changing the "token" value of the Kubernetes Secret.
```